modules/upgrading-oadp10-to12-in-mtc.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/upgrading-3-4.adoc
+// * migration_toolkit_for_containers/upgrading-mtc.adoc
+
+:_content-type: PROCEDURE
+[id="migration-upgrading-oadp-for-mtc-18_{context}"]
+= Upgrading OADP 1.0 to 1.2 for {mtc-full} 1.8.0
+
+
+.Procedure
+
+
+* For each subscription, patch the OADP operator from OADP 1.0 to OADP 1.2, by issuing:
++
+[source,terminal]
+----
+$ oc -n openshift-migration patch subscription redhat-oadp-operator-stable-1.0-mtc-operator-catalog-openshift-marketplace --type merge --patch '{"spec": {"channel":"stable-1.2"}}'
```

Comment:
Do we need to add a note or comment in the "Determine subscription..." section indicating the user-specific returned NAME values will be used for their installation of MTC & OADP, respectively?  While it will be similar, it almost certainly won't be exactly `redhat-oadp-operator-stable-1.0-mtc-operator-catalog-openshift-marketplace` we're using in this update.

Reply:
Added note